xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -189,17 +189,20 @@ bool CGUIDialogVideoInfo::OnMessage(CGUIMessage& message)
           OnSearch(directors[0]);
         else
         {
-          auto pDlgSelect = static_cast<CGUIDialogSelect*>(g_windowManager.GetWindow(WINDOW_DIALOG_SELECT));
-          pDlgSelect->Reset();
-          pDlgSelect->SetHeading(CVariant{22080});
-          for (const auto &director: directors)
-            pDlgSelect->Add(director);
-          pDlgSelect->Open();
-
-          int iItem = pDlgSelect->GetSelectedItem();
-          if (iItem < 0)
-            return true;
-          OnSearch(directors[iItem]);
+          auto pDlgSelect = dynamic_cast<CGUIDialogSelect*>(g_windowManager.GetWindow(WINDOW_DIALOG_SELECT));
```

Comment:
I've introduced a new generic method `g_windowManager.GetWindow<T>()` which do all the cast magic for us. In addition I've changed all the places where we currently use the old way. You'll need to rebase your PR.

For more info see https://github.com/xbmc/xbmc/pull/11869